gcloud-java-contrib/gcloud-java-nio/src/main/java/com/google/gcloud/storage/contrib/nio/CloudStorageFileAttributeView.java

```diff
@@ -18,11 +19,12 @@
 @Immutable
 public final class CloudStorageFileAttributeView implements BasicFileAttributeView {
 
-  private final CloudStorageFileSystemProvider provider;
+  //private final CloudStorageFileSystemProvider provider;
+  private final Storage storage;
   private final CloudStoragePath path;
 
-  CloudStorageFileAttributeView(CloudStorageFileSystemProvider provider, CloudStoragePath path) {
-    this.provider = verifyNotNull(provider);
+  CloudStorageFileAttributeView(Storage storage, CloudStoragePath path) {
+    this.storage = verifyNotNull(storage);
     this.path = verifyNotNull(path);
   }
 
```

Comment:
I see no javadoc in the snippet above your comment, but I spent the time to improve the documentation throughout by adding periods everywhere they were missing.